app/Jobs/Service/UpdateVitoAgentConfigJob.php

```
@@ -0,0 +1,73 @@
+
+class UpdateVitoAgentConfigJob implements ShouldQueue
+{
+    use Queueable;
+    use UniqueQueue;
+
+    public function __construct(protected Server $server) {}
+
+    public static function dispatchFor(Service $service): void
+    {
+        try {
+            if ($service->name === VitoAgent::id()) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Services\Monitoring\VitoAgent\VitoAgent' in method 'dispatchFor'. (undefined)

(StaticAccess)